utils/_context/_scenarios/__init__.py

```diff
@@ -337,18 +337,18 @@ class _Scenarios:
         "APPSEC_STANDALONE",
         weblog_env={
             "DD_APPSEC_ENABLED": "true",
-            "DD_EXPERIMENTAL_APPSEC_STANDALONE_ENABLED": "true",
+            "DD_APM_TRACING_ENABLED": "false",
             "DD_IAST_ENABLED": "false",
         },
         doc="Appsec standalone mode (APM opt out)",
         scenario_groups=[ScenarioGroup.APPSEC],
     )
 
-    appsec_standalone_v2 = EndToEndScenario(
-        "APPSEC_STANDALONE_V2",
+    appsec_standalone_experimental = EndToEndScenario(
+        "APPSEC_STANDALONE_EXPERIMENTAL",
```

Comment:
Renaming scenario is dangerous, you need to crawl all CI to verify that they do not refer to it, and update them if needed.

Two good enough solution for that issue : 

* do not rename scenarios : it's just names
* rename, but handle in `run.sh` legacy names , see line 378